pkg/csi/service/node.go

```diff
@@ -437,7 +437,11 @@ func (s *service) NodeUnpublishVolume(
 	}
 
 	// Figure out if the target path is a file or block volume
-	isFileMount, _ := common.IsFileVolumeMount(ctx, target, mnts)
+	isFileMount, err := common.IsFileVolumeMount(ctx, target, mnts)
```

Comment:
Wait, this is wrong! I had purposefully not returned the error here. This is NodeUnpublishVolume code, if the target path is not present in the mounts, it is not an error. You can log the error if you wish but do not return error here.

Reply:
Is it safe to assume it is not a file volume type if we could not find target path. Can we add some doc or comments for such code paths

Reply:
Sure, you can add a note here. If the target path is not present in mount paths, it can be a raw block device as well. Correct me if I am wrong here @divyenpatel 
That is the reason we check for devices in the next step and if there are no devices available, then we delete the target path and exit gracefully.